File: juros.c
#include<stdio.h>
#include<assert.h>

float juros(float capital, float taxaEmprestimo, float periodo)
{
  return capital * (taxaEmprestimo / 100) * periodo;
}

int main()
{
  // J = C.i.n -> formula
  /*
  *J = juros,
  *C = capital,
  *i = taxa de empréstimo
  *n = períodos
  */
  assert(2560 == juros(16000,4,4));
  return 0;
}

File: juros.js
const assert = require('assert');

const juros_s = (capital,taxaEmprestimo,periodo) => capital * (taxaEmprestimo / 100) * periodo;

// console.log(juros_s(16,4,4));

try {
  assert.equal(2.56,juros_s(16,4,4))
} catch (e) {
  console.log(e)
}